### DISCUSSION TITLE
SUNNY TRIPOWER 10.0 support?

### DISCUSSION THREAD

**Discussion by chrisryan10** [meltaxa/solariot#36] [Q&A]
What is required to monitor this inverter?  I've got this one installed.

Also, I'm getting another inverter installed, so how can I set up multiple inverters?

**Comment by meltaxa** [comment]
Hi @chrisryan10. There is currently no support for the Sunny Tripower. However, you could create a modbus register map if you know the values and try running Solariot. For multiple inverters, you would also have to run multiple Solariot processes, each using a different config file for each inverter.

**Reply by chrisryan10** [reply]
Ok, how do I make a modbus register map?

**Reply by meltaxa** [reply]
Firstly, you need to obtain the modbus registers for the inverter. Hopefully the vendor has documentation, otherwise with a technical background in coding you can probe the inverter (using a modbus library script such as [pymodbus](https://github.com/riptideio/pymodbus)) and matching registers with the values returned. 

Then using an existing modbus register map (for example, [modbus-sungrow-sh5k.py](https://github.com/meltaxa/solariot/blob/master/modbus-sungrow-sh5k.py)) as a template, copy and modify the map accordingly.

**Reply by meltaxa** [reply]
This forum thread might be helpful in constructing your modbus map: https://community.openenergymonitor.org/t/help-with-modbus-configuration-for-sma-tripower/9476/2.
